pkg/kubelet/dockertools/manager.go

```diff
@@ -473,6 +473,65 @@ func (dm *DockerManager) GetPodInfraContainer(pod kubecontainer.Pod) (kubecontai
 	return kubecontainer.Container{}, fmt.Errorf("unable to find pod infra container for pod %v", pod.ID)
 }
 
+// makeEnvList converts the map to a list of strings, in the form of
+// '<key>=<value>', which can be understood by docker.
+func makeEnvList(envs map[string]string) (result []string) {
+	for k, v := range envs {
+		result = append(result, k, v)
```

Comment:
I don't think this inserts with an "=" in between? But we may remove this if we change to a list.